src/core/packages/saved-objects/migration-server-internal/src/state_action_machine.ts

```diff
@@ -6,6 +6,7 @@
  * your election, the "Elastic License 2.0", the "GNU Affero General Public
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
+import apm from 'elastic-apm-node';
```

Comment:
Open to removing this APM bit for now if we feel we don't need it... Could have easily been a separate PR too